plugins/inputs/opcua/README.md

```diff
@@ -187,6 +191,64 @@ to use them.
   #   # use_unregistered_reads = false
 ```
 
+## Client Certificate Configuration
+
+When using security modes other than "None", Telegraf acts as an OPC UA client
+and requires a client certificate to authenticate itself to the server. The
+plugin supports three certificate management approaches:
+
+### Temporary Self-Signed Certificates (Default)
+
+If both `certificate` and `private_key` options are left empty or commented
+out, Telegraf will automatically generate a self-signed certificate in a
+temporary directory on each startup:
+
+```toml
+# certificate = "/etc/telegraf/cert.pem"  # commented out
+# private_key = "/etc/telegraf/key.pem"   # commented out
+```
+
+**Note:** These certificates are recreated on every Telegraf restart, requiring
+re-authorization by the OPC UA server each time. This is suitable for testing
+but not recommended for production environments.
+
+### Persistent Self-Signed Certificates (Recommended for Testing)
+
+To maintain the same client identity across restarts, specify paths for both
+`certificate` and `private_key`. If the files don't exist, Telegraf will
+generate them at the specified locations and reuse them on subsequent restarts:
+
+```toml
+certificate = "/etc/telegraf/opcua_cert.pem"
+private_key = "/etc/telegraf/opcua_key.pem"
+```
```

Comment:
I think people also do understand this without the config examples... Please remove them, also below...